src/app/routes/getInitialData/index.js

```diff
@@ -1,20 +1,12 @@
 import 'isomorphic-fetch';
 
-const getBaseUrl = () => {
-  if (process && process.env && process.env.SIMORGH_BASE_URL) {
-    // we are on the server
-    return process.env.SIMORGH_BASE_URL;
-  }
-  // we are on the client
-  const { protocol, hostname, port } = window.location;
-  return `${protocol}//${hostname}${port.length > 0 ? `:${port}` : ''}`;
-};
-
 const getInitialData = async ({ match }) => {
   try {
     const { id, service, amp } = match.params;
 
-    const url = `${getBaseUrl()}/${service}/articles/${id}.json`;
+    const url = `${
+      process.env.SIMORGH_BASE_URL
+    }/${service}/articles/${id}.json`;
```

Comment:
My browser locally is requesting http://localhost:7080/news/articles/undefined/news/articles/c85pqyj5m2ko.json here

Reply:
Thought this over on the way home. Is this a separate issue? Fix the environment variable stuff, then tackle Phil's issue in a separate PR which addresses the fact we no longer have build-time variables from Razzle? Would stop this PR becoming too big.